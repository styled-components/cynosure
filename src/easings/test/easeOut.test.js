// @flow
import easeOut from '../easeOut'

describe('easeOut', () => {
  it('should return easeOutBack cubic-bezier', () => {
    expect({
      'transition-timing-function': easeOut('back'),
    }).toMatchSnapshot()
  })

  it('should return easeOutCirc cubic-bezier', () => {
    expect({
      'transition-timing-function': easeOut('circ'),
    }).toMatchSnapshot()
  })

  it('should return easeOutCubic cubic-bezier', () => {
    expect({
      'transition-timing-function': easeOut('cubic'),
    }).toMatchSnapshot()
  })

  it('should return easeOutExpo cubic-bezier', () => {
    expect({
      'transition-timing-function': easeOut('expo'),
    }).toMatchSnapshot()
  })

  it('should return easeOutQuad cubic-bezier', () => {
    expect({
      'transition-timing-function': easeOut('quad'),
    }).toMatchSnapshot()
  })

  it('should return easeOutQuart cubic-bezier', () => {
    expect({
      'transition-timing-function': easeOut('quart'),
    }).toMatchSnapshot()
  })

  it('should return easeOutQuint cubic-bezier', () => {
    expect({
      'transition-timing-function': easeOut('quint'),
    }).toMatchSnapshot()
  })
})
