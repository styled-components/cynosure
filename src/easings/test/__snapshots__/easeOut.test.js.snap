// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`easeOut should return easeOutBack cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.175,  0.885, 0.320, 1.275)",
}
`;

exports[`easeOut should return easeOutCirc cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.075,  0.820, 0.165, 1.000)",
}
`;

exports[`easeOut should return easeOutCubic cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.215,  0.610, 0.355, 1.000)",
}
`;

exports[`easeOut should return easeOutExpo cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.190,  1.000, 0.220, 1.000)",
}
`;

exports[`easeOut should return easeOutQuad cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.250,  0.460, 0.450, 0.940)",
}
`;

exports[`easeOut should return easeOutQuart cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.165,  0.840, 0.440, 1.000)",
}
`;

exports[`easeOut should return easeOutQuint cubic-bezier 1`] = `
Object {
  "transition-timing-function": "cubic-bezier(0.230,  1.000, 0.320, 1.000)",
}
`;
